src/Conversion/ONNXToKrnl/NN/Conv.cpp

```diff
@@ -36,8 +36,10 @@ struct ONNXConvNoBiasOpLowering : public ConversionPattern {
     auto inputShape = inputOperand.getType().cast<MemRefType>().getShape();
     auto &kernelOperand = operands[1];
     auto kernelShape = kernelOperand.getType().cast<MemRefType>().getShape();
+    auto &biasOperand = operands[2];
+    bool hasBias = !biasOperand.getType().isa<NoneType>();
 
```

Comment:
We can get an operand by its index (like here) or by name. Do we have an preference/rule for our code?

Reply:
I think we should always go by the name because the code would be more readable, and in case it is impossible to address a specific operand by name (as in the case for variadic operation), we should go by index as a fall-back solution.

Reply:
By casting the operands to the operandAdaptor, I was able to use the real names. It worked, please check that it looks right to you too, and then we can make a ticket that clean all these numbers away